tiles-request-api/src/test/java/org/apache/tiles/request/locale/URLApplicationResourceTest.java

```diff
@@ -201,4 +224,36 @@ public void testGetInputStreamWithSpace() throws IOException {
     	assertNotNull(is);
     	is.close();
     }
+
+    @Test
+    public void testLocalProtocol() throws IOException {
+        URL url = new URL("test1://foo/bar.txt");
+        URLApplicationResource resource = new URLApplicationResource("org/apache/tiles/request/test/locale/resource.txt", url);
+        try {
+            resource.getInputStream();
+        } catch (FileNotFoundException e) {
+            // expected
```

Comment:
if this is expected, then shouldn't the method be annotated `@Test(expected=FileNotFoundException.class)` ?